src/main/kotlin/it/pagopa/wallet/eventdispatcher/service/WalletCDCService.kt

```diff
@@ -0,0 +1,26 @@
+package it.pagopa.wallet.eventdispatcher.service
+
+import it.pagopa.wallet.eventdispatcher.audit.LoggingEvent
+import java.util.*
+import org.slf4j.LoggerFactory
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.stereotype.Component
+import reactor.core.publisher.Mono
+
+@Component
+class WalletCDCService(
+    private val cdcKafkaTemplate: KafkaTemplate<String, Any>,
```

Comment:
Starting from [this](https://github.com/pagopa/pagopa-payment-wallet-event-dispatcher-service/commit/ecbffd9e01c6df7744f908735864bf4282884708) modification this should be 
```suggestion
    private val cdcKafkaTemplate: ReactiveKafkaProducerTemplate <String, Any>,
```

Reply:
yes this was already edited, check last commits :)